components/body/desktop.styl

```diff
@@ -72,20 +61,24 @@
 
   .vertical-rectangle
     height 280px
-  .column-one
-    margin-right 10px
-  .column-two
-    padding-top 150px
-    margin-left 10px
 
-  .column-one img
-    padding-top 0
+  .column
+    margin 0 5px
 
-  .column-two
-    maigin-top 0px
+  .staggered
+    padding-top 150px
 
   .section-two-text-container
     position relative
     top 50%
     transform translateY(130%)
 
+  #partnership-header.is-desktop
+    display block
+    border-bottom-style none
+    font-size 24px
+    text-align center
+    font-weight bolder
+
+  #remove-top-padding
+    padding-top 0px
```

Comment:
It was previously a class but the rule wasn't being applied and I also forgot I used it more than once.